src/app/dashboard/competitions/[competitionId]/participants/components/ParticipantsManage.tsx

```diff
@@ -95,15 +100,22 @@ export default function ParticipantsManage({ participants }: ParticipantsManageP
             {pendingParticipants.map(participant => (
               <tr key={participant.id}>
                 <td>
-                  {Object.entries(participant.competitionsInformation.userInformation as Record<string, string>).map(([key, value]) => (
-                    <p key={key}>{key}: {value}</p>
-                  ))}
+                  {Object.entries(participant.competitionsInformation.userInformation as Record<string, string>)
+                    .filter(([key]) => key !== 'paymentFile')
+                    .map(([key, value]) => (
+                      <p key={key}>{key}: {value}</p>
+                    ))}
+                  <p>Rut: {formatRutToShow(participant.rut)}</p>
                 </td>
                 <td>
                   <p>{participant.competitionsInformation.category}</p>
                 </td>
                 <td>
-                  <button className="btn btn-info btn-sm mr-2">Ver datos de inscripción</button>
+                  {
+                    paymentsFileUrls.find(payment => payment.participantId === participant.id)?.paymentFileUrl && (
+                      <a href={paymentsFileUrls.find(payment => payment.participantId === participant.id)?.paymentFileUrl || ''} target="_blank" className="btn btn-info btn-sm mr-2">Ver comprobante de pago</a>
+                    )
```

Comment:
Esto es ineficiente ya que estás haciendo 2 consultas a la base de datos:
- Una vez para verificar si existe el comprobante de pago
- Otra vez para obtener la URL del comprobante

```suggestion
                    (() => {
                      const paymentFile = paymentsFileUrls.find(payment => payment.participantId === participant.id)?.paymentFileUrl;
                      return paymentFile && (
                        <a href={paymentFile} target="_blank" className="btn btn-info btn-sm mr-2">Ver comprobante de pago</a>
                      );
                    })()
```